client/my-sites/email/gsuite-add-users/index.jsx

```diff
@@ -261,31 +262,29 @@ class GSuiteAddUsers extends React.Component {
 		const googleMailServiceFamily = getGoogleMailServiceFamily( getProductSlug( productType ) );
 
 		return (
-			<Fragment>
-				<PageViewTracker path={ analyticsPath } title="Domain Management > Add G Suite Users" />
+			<>
+				<PageViewTracker path={ analyticsPath } title="Email Management > Add New Users" />
 
 				{ selectedSite && <QuerySiteDomains siteId={ selectedSite.ID } /> }
+
 				{ selectedSite && <QueryGSuiteUsers siteId={ selectedSite.ID } /> }
 
-				<Main>
-					<DomainManagementHeader
-						onClick={ this.goToEmail }
-						selectedDomainName={ selectedDomainName }
-					>
-						{ googleMailServiceFamily }
-					</DomainManagementHeader>
-
-					<EmailVerificationGate
-						noticeText={ translate( 'You must verify your email to purchase %(productFamily)s.', {
-							args: { productFamily: googleMailServiceFamily },
-							comment: '%(productFamily)s can be either "G Suite" or "Google Workspace"',
-						} ) }
-						noticeStatus="is-info"
-					>
-						{ this.renderAddGSuite() }
-					</EmailVerificationGate>
-				</Main>
-			</Fragment>
+				<DocumentHead title={ translate( 'Add New Users' ) } />
+
+				<EmailHeader currentRoute={ currentRoute } selectedSite={ selectedSite } />
+
+				<HeaderCake onClick={ this.handleBack }>{ translate( 'Add new users' ) }</HeaderCake>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **145** times:
`translate( 'Add New Users' )` *ES Score: 7*